src/server/config/express.js

```diff
@@ -55,6 +56,14 @@ export default (server) => {
     proxyOneQuran.web(req, res);
   });
 
+  server.use('/apple-app-site-association', (req, res) => {
```

Comment:
can we set the content-type from the static folder? the file cannot have an extension of `.json`